subworkflows/local/prepare_references.nf

```diff
@@ -153,6 +162,7 @@ workflow PREPARE_REFERENCES {
         gnomad_af_idx         = TABIX_GNOMAD_AF.out.tbi.collect()                                // channel: [ val(meta), path(fasta) ]
         known_dbsnp_tbi       = TABIX_DBSNP.out.tbi.collect()                                    // channel: [ val(meta), path(fasta) ]
         target_bed            = Channel.empty().mix(ch_tbi, ch_bgzip_tbi).collect()              // channel: [ val(meta), path(bed), path(tbi) ]
+        vcfanno_extra         = ch_vcfanno_extra.ifEmpty([[]])                                   // channel: [ path(intervals) ]
```

Comment:
Please update this comment

Reply:
Sorry! I did change this, but don't know why the site didn't pick it up. Perhaps I changed the page too soon 😬

Reply:
no worries! 😄